frontend/types/charity.ts

```diff
@@ -1,3 +1,21 @@
-export type Charity = {
+import { WithoutId } from './utils';
+import { CampaignPostData } from './campaigns';
+
+export type CharityData = CharityListData & {
+  description: string;
+  websiteUrl: string;
+  imageBase64: string;
+};
+
+export type CharityListData = CharityMinimalData & {
+  logoBase64: string;
+};
+
+export type CharityMinimalData = {
   id: number;
+  name: string;
 };
```

Comment:
Do you mind reordering the types such that they are in order of dependency? So that we look upwards to find the dependent types